config/environments/production.rb

```diff
@@ -24,7 +24,11 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
we shouldn't because we don't have a js engine in production ... but I don't know if that is something required for puma to work...@miquel: is it?